### PR TITLE
feat(platform): add getOnboardingThemes query

### PR DESCRIPTION
**Pull request by lovethebomb**
- Add query `getOnboardingThemes`
- Fix Platform.chapters.list api version, v1 -> v2

Usage

```js
const [themes] = invoke(getOnboardingThemes, {})
```

```json
{
            "meta": {
                "createdAt": "2021-09-06T12:03:54.245Z",
                "updatedAt": "2021-09-10T16:16:48.600Z"
            },
            "hidden": false,
            "_id": "613603aa1d3dea85870f54b9",
            "ref": "ALL",
            "__v": 0,
            "description": "ALL.description",
            "name": "Tous les cours",
            "type": "theme",
            "repository": "content-catalogue@cockpit-catalog",
            "courseCount": 288,
            "chapterCount": 1835,
            "externalCourseCount": 3,
            "id": "613603aa1d3dea85870f54b9",
            "cover": "//static.coorpacademy.com/content/CoorpAcademy/content-eyrolles/cockpit-eyrolles/default/eyrolles_nego-complexe_1a3-1504094597067.jpg"
        },
```

Right now we still hit the theme `ALL` which fetches 1800+ chapters, but this should be fixed on the CoorpAcademy API side soon

### CODE REVIEW

File: app/onboarding/queries/getOnboardingThemes.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import { resolvePlatform } from "lib/resolvers"
+import Platform from "lib/coorpacademy/platform"
+
+// TODO: Separate token from platform to upskill
+const UPSKILL_PLATFORM_ID = "internal-upskill"
+const UPSKILL_PLATFORM_TOKEN = process.env.COORPACADEMY_TOKEN
+
+export default resolver.pipe(async (_, { session }) => {
+  // TODO: Replace with user session corp token?
```
Comment (alanlanglois) [inline app/onboarding/queries/getOnboardingThemes.ts:11]: I'm not sure we'll use the user token during the onboarding?

File: app/onboarding/queries/getOnboardingThemes.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    token: UPSKILL_PLATFORM_TOKEN,
+  })
+  let request = await upskillPlatform.groups.list()
+  const upskillThemes = await request.json()
+
+  // TODO: We mock the cover of the themes with the first theme, first course cover
+  request = await upskillPlatform.chapters.listByTheme(upskillThemes[0].ref)
+  const upskillChapters = await request.json()
+
+  const cover = upskillChapters[0].poster.mediaUrl
```
Comment (alanlanglois) [inline app/onboarding/queries/getOnboardingThemes.ts:24]: Add a fallback when upskillChapter[0].poster.mediaUrl is empty (or upskillChapters[0] null)

File: app/onboarding/queries/getOnboardingThemes.ts (hunk 1 above)
Comment (lovethebomb) [reply app/onboarding/queries/getOnboardingThemes.ts:11]: Woops copy pasta, resolved in #36 🙈

File: app/onboarding/queries/getOnboardingThemes.ts (hunk 2 above)
Comment (lovethebomb) [reply app/onboarding/queries/getOnboardingThemes.ts:24]: Added FALLBACK_COVER support in #36

**Pull request merged**